### ISSUE TITLE
PLANET-5207 Articles Block: Ensure proper reading by Screen Reader

### ISSUE THREAD

**Issue by planet-4** [greenpeace/planet4#83]
Used the ScreenReader (NVDA, VoiceOver) to test the [Articles Block](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/master/assets/src/blocks/Articles/ArticlePreview.js) and check the reading flow makes sense and has no glitches or unwanted texts.

Problems found so far:
 * The mid dot separating the author and the date should be invisible to the screen reader (that might be already taken care of, see [this ticket](https://jira.greenpeace.org/browse/PLANET-5256)).
 * The text "by" preceding the author is not part of the link, so it's read separately by the Screen Reader, making a pause. It should be part of the link.
 * The image caption is read but it doesn't say it's an image. Before the caption we should add something like "This is an image, the caption says: ".

**Tasks:**
* Fix the problems mentioned above
* Attach a video with the result to this task

**Notes:**
For the pseudo element we might be able to solve this with [media queries](https://stackoverflow.com/questions/26634156/can-i-prevent-after-pseudo-element-from-being-read-by-screen-readers) so we don't have to change the markup.

```
Reporter: pcuadrad
Sections: Block: Articles
```
Potentially affected repositories: [planet4-plugin-gutenberg-blocks](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks)

### Testing
You can add the Articles block to a page and test it with a screen reader, the block should also already be present on your local [homepage](http://www.planet4.test/) so you can use that too. You can check the [docs](https://planet4.greenpeace.org/act/articles/) for more details on how the block works.
